### DISCUSSION TITLE
Which font to use?

### DISCUSSION THREAD

**Discussion by KoB-Kirito** [stefanhaustein/TerminalImageViewer#124] [Q&A]
Hey, nice app, but I got some problems with the font.

I'm using Windows Powershell, and there Lucida Console is the default. Pretty, but almost none of the needed chars there.

I installed Inconsolata and Fira Code as you suggested, but:
Inconsolata is quite ugly (in my opinion) and has weird spacing and misalignments in the images.
Fire Code is nice, but some chars are missing (I'm getting some ? in the images).

So what font would you suggest?

**Comment by aaronliu0130** [comment]
I'd suggest San Francisco Mono(Mac pkg here<https://developer.apple.com/fonts/>, otf files here<https://github.com/supercomputra/SF-Mono-Font>), Hack<https://sourcefoundry.org/hack/>, and Ubuntu Mono<https://design.ubuntu.com/font/>.
________________________________

**Comment by clort81** [comment]
Almost no fonts render the traingular glyphs correctly.  They do so incorrectly.

Unscii renders them correctly.
http://pelulamu.net/unscii/

Also Microsoft monospace fonts tend to miss some needed block chars.

**Comment by stefanhaustein** [comment]
I think we have disabled the triangular glyphs for that reason.
